website/docs/language/backend/s3.mdx

```diff
@@ -157,19 +175,19 @@ data.terraform_remote_state.network:
 
 ## Configuration
 
-This backend requires the configuration of the AWS Region and S3 state storage. Other configuration, such as enabling DynamoDB state locking, is optional.
+This backend requires the configuration of the AWS Region and S3 state storage. Other configuration, such as enabling state locking, is optional.
 
 ### Credentials and Shared Configuration
 
 !> **Warning:**  We recommend using environment variables to supply credentials and other sensitive data. If you use `-backend-config` or hardcode these values directly in your configuration, Terraform will include these values in both the `.terraform` subdirectory and in plan files. Refer to [Credentials and Sensitive Data](/terraform/language/backend#credentials-and-sensitive-data) for details.
 
 The following configuration is required:
 
-* `region` - (Required) AWS Region of the S3 Bucket and DynamoDB Table (if used). This can also be sourced from the `AWS_DEFAULT_REGION` and `AWS_REGION` environment variables.
+* `region` - (Required) AWS Region of the S3 Bucket and DynamoDB Table (if used, **deprecated**). This can also be sourced from the `AWS_DEFAULT_REGION` and `AWS_REGION` environment variables.
```

Comment:
nit - I think the deprecation is covered well in other sections. Adding it here makes the sentence a bit harder to understand, so I'd propose reverting to the original syntax.

```suggestion
* `region` - (Required) AWS Region of the S3 Bucket and DynamoDB Table (if used). This can also be sourced from the `AWS_DEFAULT_REGION` and `AWS_REGION` environment variables.
```